Header header

int16 idx1
int16 idx2
